charts/opentelemetry-collector/examples/deployment-with-protected-loadBalancer/values.yaml

```diff
@@ -0,0 +1,9 @@
+mode: deployment
```

Comment:
These are simple enough fields that I don't think we need a new example to demonstrate them